CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.1.7
```

Comment:
please put the `-dev` here too so it matches the pubspec version

Reply:
We're doing that now? Was there a "discussion" on this?

Reply:
I have always argued that they should always be in sync 🤷‍♂ . I am not aware of any agreed upon policy that is consistently used across repos though.